bedrock/firefox/templates/firefox/android/index.html

```diff
@@ -61,6 +61,11 @@ <h2 class="heading">
           <a href="{{ url('firefox.sms') }}">Text a download link to my Android phone</a>
         </p>
       </div>
+
+      {% if l10n_has_tag('ios_announce', 'firefox/new') %}
+        <p class="ios-link">{{_('Firefox is coming soon to iOS! <a href="{url}">Sign up to learn more</a>.').format(url=url('newsletter.ios'))|safe }}</p>
```

Comment:
Please don't change the string at this point, it's already been exposed for localization. You can also `display: block` the anchor.